LazaroProject/LazaroCuis
# Prevent duplicate processes

## Problem
Currently ScreenReader speaks when it receive the message `#say: aText`, but sometimes it could receive multiple times that message and the processes could overlap.

## Solution
Save the current process before run it and kill the current one before replace with the new one.

## How to test it
Open the menu word and select the items fast, no sound should overlap another.

Merge it after https://github.com/LazaroProject/LazaroCuis/pull/2
